src/content/docs/more-integrations/open-source-telemetry-integrations/dbt-cloud/dbt-cloud-integration.mdx

```diff
@@ -0,0 +1,373 @@
+---
+title: Dbt Cloud with Airflow and Snowflake Integration 
+tags:
+    - New Relic integrations
+    - Dbt Cloud integrations 
+redirects: 
+  - /docs/install/dbt-cloud-integration
+metaDescription: Dbt Cloud with Airflow and Snowflake integration 
+freshnessValidatedDate: never
+---
+
+Our Dbt cloud integration with Airflow monitors the health of your dbt Cloud jobs
+and resources, helping you identify problems like failed runs, failed models and failed tests. 
+This integration runs on Apache Airflow and queries Snowflake for any failed tests if configured to do so.
+
+## Prerequisites
+* Dbt Cloud account with APIs enabled and using Snowflake as the database. 
+* Snowflake account where the dbt Cloud account runs on.
+* Existing Airflow environment >= 2.8.1 or ability to run docker compose.
+* New Relic account.
```

Comment:
I notice this is in our Open Telemtry category. Are there any OTEL/new relic requirements? how does otel fit into this?

Reply:
Oh shoot. No, this should not be OTEL. I'll move it.